### DISCUSSION TITLE
Customizing "about" templates (website)

### DISCUSSION THREAD

**Discussion by kcarnold** [quarto-dev/quarto-cli#4580] [Q&A] [answered]
If I like the general layout of one of the "about" pages but want to customize one little thing, can I extend the builtin templates or am I stuck rebuilding all the functionality?

If I try to copy one of the template `ejs` files and specify it as the `template`, I get an error ("ERROR: Validation of YAML front matter failed."). https://github.com/quarto-dev/quarto-cli/blob/6dcadb85d19dbee547f8baaec45b7b9f1dd16a99/src/resources/schema/definitions.yml#L1188 seems to be pretty picky.

Sorry if this has been discussed before; "about" is hard to search for.

**Comment by dragonstyle** [comment]
Our next pre-release build will support passing a path to a template in the `template` param. We haven't documented what is available in the template schema, so your mileage may vary, but this at least allow you to end the template through.

Note that if you're copying the built in in templates, they have expectations about the files prefixed with `_` being available in the same directory.

**Reply by jonasjuerss** [reply]
I assume this has been implemented by now. Unfortunately, I couldn't find any documentation. Would you mind linking it here?

**Reply by mcanouil** [reply]
See https://quarto.org/docs/websites/website-about.html#templates

**Reply by jonasjuerss** [reply]
I don't think this says anything about custom templates. I was able to pass a path to a local template file, but some documentation on creating them would be a nice to have.

**Reply by mcanouil** [reply]
- https://github.com/quarto-dev/quarto-cli/tree/e426fbc64d32af1b607e0ab3d8d55e05229c5a2f/src/resources/projects/website/about

Based on the templates, it's using EJS templating system.

**Reply by mcanouil** [reply]
It seems the documentation was not updated based on this discussion.
I'm opening a documentation issue.
Thanks.

**Reply by mcanouil** [reply, marked as answer]
Too late, @cderv opened one ^^

- https://github.com/quarto-dev/quarto-cli/issues/12790
